### ISSUE TITLE
CE limit

### ISSUE THREAD

**Issue by minijaham** [DaPigGuy/PiggyCustomEnchants#273]
### Description
Duplicate of #261 , but I wanted to make my point clear. I am currently the owner of a server, which I am currently developing on and I was looking for a custom enchantment plugin because I did not want to code a whole different CE core .-.(great plugin btw, I love it)

I definitely need a feature that which sets a limit on the number of CEs that can be enchanted on an item(can be configured in config), because having more than 8 CEs make the item very overpowered, even the working chances of the enchantments are lowered down in the config.

If you can't update the plugin furthermore with this feature being added, please consider at least telling me how to try to add the feature to the plugin(by coding extra stuff in) because I've tried it multiple times and all of them...yeah I messed it up lol

Thanks! Also be careful of corona!

**Comment by Thunder33345** [comment]
this should be done on your enchanting/enchant shop plugin

**Comment by minijaham** [comment]
> this should be done on your enchanting/enchant shop plugin

Yes, but I want most of the CEs that PiggyCustomEnchants support :(

**Comment by Thunder33345** [comment]
what i mean is the limit should be configured in the enchanting plugin
example, enchant sign, has a limit of 3, that means it check for 3 ce, if it has more, dont enchant
it doesn't make any architectural sense sense for this limit to be done in the CE
what if i as a dev wanna add more?
what if i wanna overwrite it?
this doesnt make sense
